### PR TITLE
Step1 : 학습 테스트 실습

### PR DESCRIPTION
**Pull request by devyonghee**
안녕하세요 리뷰어님!
우아한 테크 캠프 프로에 참여하게 된 한용희입니다.
앞으로 잘 부탁드리겠습니다.
감사합니다.

### CODE REVIEW

File: src/test/java/study/SetTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	void size() {
+		//when
+		int size = numbers.size();
+
+		//then
+		assertThat(size)
+			.isEqualTo(3);
+	}
+
+	@ParameterizedTest(name = "{displayName}[{index}] {0} is contained in the set")
```
Comment (na-mul) [inline src/test/java/study/SetTest.java:40]: Display Name을 잘 다듬어주신 부분 👍입니다. 
`ParameterizedTest`도 결국 각각이 개별적인 테스트이기에, 의미있는 이름이 부여되는 것이 좋다고 생각해요.

File: src/test/java/study/StringTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		//then
+		assertThat(characterByIndex)
+			.isEqualTo(expected);
+	}
+
+	@Test
+	@DisplayName("문자 길이를 벗어난 위치로 검색하면 StringIndexOutOfBoundsException 발생")
+	void charAt_outIndex_thrownIndexOutOfBoundsException() {
+		//given, when
+		ThrowingCallable callCharAt = () -> "abc".charAt(Integer.MAX_VALUE);
```
Comment (na-mul) [inline src/test/java/study/StringTest.java:66]: 매직 넘버를 사용하지 않는 것은 좋은 방향성이지만, 
그걸 감수하고라도 edge case에 대한 테스트를 하는 것도 괜찮아보여요!

**Review by na-mul — APPROVED** [review]
안녕하세요. 용희님의 첫번째 리뷰를 맡게 되어 영광입니다.
꼼꼼한 학습 테스트 잘 봤습니다!
이어지는 다음 단계도 잘 부탁드립니다🙇‍♂️

File: src/test/java/study/SetTest.java (hunk 1 above)
Comment (devyonghee) [reply src/test/java/study/SetTest.java:40]: 감사합니다! 😆
ParameterizedTest  default 이름으로는 테스트를 알아보는 것이 쉽지 않아 추가하게 됐습니다 😄

File: src/test/java/study/StringTest.java (hunk 2 above)
Comment (devyonghee) [reply src/test/java/study/StringTest.java:66]: 감사합니다 😄
